packages/sandbox/src/app/sandbox/docs/card-carousel.doc.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, CardCarousel } from '@getluko/streamline';
+import React from 'react';
+
+export const CardCarouselSandbox = () => {
```

Comment:
You can use the `DocList` component for your `.doc` (e.g `card-highlight.doc.tsx`)

Reply:
Replaced by the `<DocList ../>` component